plugins/in_windows_exporter_metrics/we_wmi.c

```diff
@@ -150,6 +150,32 @@ static int wmi_update_counters(struct wmi_query_spec *spec, uint64_t timestamp,
     return 0;
 }
 
+static char *convert_prop_to_str(VARIANT *prop, int handle_null)
+{
+    char *strlabel = NULL;
+    char *newstr = NULL;
+
+    if (handle_null == FLB_TRUE && prop->vt == VT_NULL) {
+        newstr = strdup("");
+        if (newstr == NULL) {
+            return NULL;
+        }
+    }
+    else {
+        VariantChangeType(prop, prop, 0, VT_BSTR);
```

Comment:
Please check the result of VariantChangeType, it can fail according to the [documentation](https://learn.microsoft.com/en-us/windows/win32/api/oleauto/nf-oleauto-variantchangetype#return-value)

Reply:
Indeed. Fixed in https://github.com/fluent/fluent-bit/pull/7395/commits/a9833f56268141ff39b55eed3ddb5d3229ecf5cb.